lib/rika.rb

```diff
@@ -4,7 +4,7 @@
 
 require "rika/version"
 require 'uri'
-require 'net/http'
+require 'open-uri'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.